images/dockerregistry/config.yml

```diff
@@ -35,3 +35,8 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
+  version: 1.0
+  metrics:
+    enabled: false
+    secret: TopSecretToken
```

Comment:
This is incredibly unsafe and needs to be fixed :)

We should not check in default secrets - instead this should be empty, and the user MUST set both secret and enabled in order to turn it on.  That should be done by `oc adm registry`